contents/tutorials/android-analytics.md

```diff
@@ -426,4 +426,4 @@ That's it! Feel free to play around in your dashboard and explore the different
 
 - [How to run A/B tests in Android](/tutorials/android-ab-tests)
 - [How to set up feature flags in Android](/tutorials/android-feaure-flags)
-- [How to run A/B tests in iOS](/tutorials/ios-ab-tests)
+- [How to set up session replays in Android](/tutorials/android-session-replays)
```

Comment:
should we just add a new item instead of replacing it?

Reply:
we usually stick to 3. Also the other one is iOS, so I think android is more relevant (for google ranking purposes)